Library/Homebrew/test/dev-cmd/cat_spec.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.describe "brew cat" do
   it_behaves_like "parseable arguments"
 
-  it "prints the content of a given Formula", :integration_test do
+  it "prints the content of a given Formula", :integration_test, :needs_network do
```

Comment:
Why does this need the network? 🤔

Reply:
`brew cat` will attempt to download the formula/cask source from Github Pages if it's not available locally.